docs/integrations.md

```diff
@@ -34,53 +34,58 @@ Zafira uses integration with Amazon S3 service to store user profile photos and
 * Grant read/write permissions for new user (read [detailed guide](https://aws.amazon.com/blogs/security/writing-iam-policies-how-to-grant-access-to-an-amazon-s3-bucket/))
```

Comment:
Grant read / write permissions for the new user (see [Detailed Guide] (https://aws.amazon.com/blogs/security/writing-iam-policies-how-to-grant-access-to-an- amazon- s3-bucket /))